internal/export/database/export.go

```diff
@@ -32,8 +34,18 @@ const (
 	oneDay       = 24 * time.Hour
 )
 
+type ExportDB struct {
+	db *database.DB
+}
+
+func NewExportDB(db *database.DB) *ExportDB {
```

Comment:
Will this eventually take additional params, or use a default for db if the db is nil? If so what do you think about using an option pattern:

```
func ExportDB(opts ...ExportDBOption) *ExportDB {
...
}

func WithDB(db *DB) ExportDBOption {
}
```

Reply:
Unsure at this point if we will need more options. Thanks for the pattern though!